src/operator/nn/mkldnn/mkldnn_base-inl.h

```diff
@@ -1,22 +1,3 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one
- * or more contributor license agreements.  See the NOTICE file
- * distributed with this work for additional information
- * regarding copyright ownership.  The ASF licenses this file
- * to you under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance
- * with the License.  You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-
```

Comment:
Although this file is licensed under ASF 2.0, it doesn't have the exact same header that we look for in `tools/license_header.py`, so I think we need to add it to the whitelist here: 
https://github.com/apache/incubator-mxnet/blob/v1.9.x/tools/license_header.py#L67

Reply:
added mkldnn_base-inl.h to the whilelist under "Licensed under Apache 2.0 license"